packages/server/db.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+
+// Подключение к MongoDB
+mongoose.connect('mongodb+srv://thefirstone.spi67la.mongodb.net/', {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+
+// Обработка ошибок подключения
+mongoose.connection.on('error', (error) => {
+  console.error('MongoDB connection error:', error);
+});
+
+mongoose.connection.once('open', () => {
+  console.log('Connected to MongoDB');
+});
+
+// Определение схемы и модели данных
```

Comment:
If we need comments change please to English language, the same for buttons label etc